rules/exposure-to-internet-via-istio-ingress/rule.metadata.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "exposure-to-internet-via-istio-ingress",
+  "attributes": {
+    "useFromKubescapeVersion": "v3.0.9"
+  },
+  "ruleLanguage": "Rego",
+  "match": [
+    {
+      "apiGroups": [
+        ""
+      ],
+      "apiVersions": [
+        "v1"
+      ],
+      "resources": [
+        "Pod",
+        "Service"
+      ]
+    },
+    {
+      "apiGroups": [
+        "apps"
+      ],
+      "apiVersions": [
+        "v1"
+      ],
+      "resources": [
+          "Deployment",
+          "ReplicaSet",
+          "DaemonSet",
+          "StatefulSet"
+      ]
+    },
+    {
+      "apiGroups": [
+        "batch"
+      ],
+      "apiVersions": [
+        "*"
+      ],
+      "resources": [
+          "Job",
+          "CronJob"
+      ]
+    },
+    {
+      "apiGroups": [
+        "networking.istio.io"
+      ],
+      "apiVersions": [
+        "v1"
+      ],
+      "resources": [
+        "*"
+      ]
```

Comment:
@amirmalka @matthyx This is a problematic line IMHO

The problem is if I put here the specific resource kinds like `Gateway` or `VirtualService` the rego does not work with Kubescape (while it works with the unit test). Any idea?

Reply:
I found the problem; needed to put `Gateways` instead of `Gateway`. But `VirtualService` is ok 😮‍💨